docs/experimental-features/investigation-guide-actions.asciidoc

```diff
@@ -0,0 +1,80 @@
+[[interactive-investigation-guides]]
+== Interactive investigation guides
+
+NOTE: This feature is available for {stack} versions 8.6.0 and newer.
+
+Detection rule investigation guides suggest steps for triaging, analyzing, and responding to potential security issues. For custom rules, you can create an interactive investigation guide that includes buttons for launching runtime queries in <<timelines-ui,Timeline>>, using both alert data and hard-coded literal values. Each query button displays the number of documents found.
```

Comment:
Combining the first sentence in the next para with this para because it feels a little disjointed. 
```suggestion
Detection rule investigation guides suggest steps for triaging, analyzing, and responding to potential security issues. For custom rules, you can create an interactive investigation guide that includes buttons for launching runtime queries in <<timelines-ui,Timeline>>, using alert data and hard-coded literal values. Each query button displays the number of documents found, which allows you to launch detailed Timeline investigations directly from an alert using relevant data.
```